semantic.d

```diff
@@ -11010,12 +11023,16 @@ mixin template Semantic(T) if(is(T==ReferenceAggregateDecl)){
 			}
 		}
 	Lfresh:
-		if(!set && decl.stc & STCoverride){
+		if(set) goto Lfresh3;
```

Comment:
Updated according to your suggestion (I didn't do it myself because I was unable to figure out the type of `set` ;-)).
Unfortunately the additional scope has to stay because if the `setnr` variable.